src/test/lifecycle/common.js

```diff
@@ -358,6 +358,36 @@ function id_lifecycle_configuration(Bucket, Key) {
     };
 }
 
+function duplicate_id_lifecycle_configuration(Bucket, Key) {
+    const ID1 = 'rule_id';
+    const ID2 = 'rule_id';
```

Comment:
Just for it to be super clear I would change it to:
```suggestion
    const ID2 = ID1; // set duplicate id
```

Reply:
Sure, thanks